cfgov/unprocessed/js/modules/multi-select.js

```diff
@@ -0,0 +1,494 @@
+'use strict';
+
+function init() {
+  var _ = function ( elem, opts ) {
+    var self = this;
+
+    this.elem = $( elem );
+    opts = opts || {};
+
+    configure.call( this, {
+      minChars:  3,
+      maxItems:  10,
+      autoFirst: false,
+      filter: _.FILTER_CONTAINS
+    }, opts );
+
+    this.index = -1;
+    this.name = this.elem.name;
+    this.skipBlur = false;
+    this.optionData = [];
+    this.selections = [];
+    this.options = this.elem.options || [];
+    this.placeholder;
+
+    _.all.push( this );
+  };
+
+  _.prototype = {
+    set options( options ) {
```

Comment:
`set` is not supported in IE8. It needs a polyfill or transpiler (such as babel).